### PR TITLE
Add cellar configuration for Active Storage (Ruby On Rails)

### PR DESCRIPTION
**Pull request by 16**

### CODE REVIEW

File: commons/addons/cellar.md [hunk 1]
```diff
@@ -195,3 +195,30 @@ see doc : http://docs.pythonboto.org/en/latest/ref/s3.html#boto.s3.bucket.Bucket
...
+  service: S3
+  access_key_id: <%= ENV.fetch('CELLAR_ADDON_KEY_ID') %>
+  secret_access_key: <%= ENV.fetch('CELLAR_ADDON_KEY_SECRET') %>
+  endpoint: https://<%= ENV.fetch('CELLAR_ADDON_HOST') %>
+  region: 'us-west-1'
+  force_path_style: true
+  bucket: mybucket
+```
+
+A `region` parameter must be provided, although it is not used by Cellar.
```
Comment (vvenance) [inline commons/addons/cellar.md]: can you explain why or make that point more clear ? seems weird to me like that

File: commons/addons/cellar.md [hunk 2]
```diff
@@ -195,3 +195,30 @@ see doc : http://docs.pythonboto.org/en/latest/ref/s3.html#boto.s3.bucket.Bucket
...
+
+`force_path_style` must be set to `true` as described in the [Ruby S3 Client documentation](https://docs.aws.amazon.com/sdkforruby/api/Aws/S3/Client.html):
+
+> `:force_path_style` (Boolean) — default: `false` —
```
Comment (vvenance) [inline commons/addons/cellar.md]: I don't get why it isn't setted to true regarding what's just being said

**Comment by cnivolle** [comment]
Hi @16 , did you had time to check on @vvenance comments ?
Thanks,

File: commons/addons/cellar.md (hunk 2 above)
Comment (16) [reply commons/addons/cellar.md]: Cellar bucket name must be left in the request URI, never moved to the host as a subdomain. So `force_path_style` must be set to `true` as described in the [Ruby S3 Client documentation](https://docs.aws.amazon.com/sdkforruby/api/Aws/S3/Client.html#constructor_details).

File: commons/addons/cellar.md [hunk 3]
```diff
@@ -195,3 +195,30 @@ see doc : http://docs.pythonboto.org/en/latest/ref/s3.html#boto.s3.bucket.Bucket
...
+> When set to true, the bucket name is always left in the request URI and never moved to the host as a sub-domain.
```
Comment (16) [inline commons/addons/cellar.md]: So we can remove this line

File: commons/addons/cellar.md (hunk 1 above)
Comment (vvenance) [reply commons/addons/cellar.md]: perfect thank you

File: commons/addons/cellar.md (hunk 2 above)
Comment (vvenance) [reply commons/addons/cellar.md]: same

**Pull request merged**